src/utils/gatewayCreateEditHelpers.test.ts

```
@@ -0,0 +1,102 @@
+import { generateUniqueId, removeCertsAndTlsOptionsForPassthrough } from './gatewayCreateEditHelpers';
```

ESLint

[error] Replace `·generateUniqueId,·removeCertsAndTlsOptionsForPassthrough·` with `⏎··generateUniqueId,⏎··removeCertsAndTlsOptionsForPassthrough,⏎`

(prettier/prettier)